.github/workflows/compatibility.yml

```diff
@@ -0,0 +1,38 @@
+name: PHP Compatibility
+
+on: ['workflow_call']
+
+jobs:
+  php-compatibility:
+    name: PHP ${{ matrix.php-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ["7.4", "8.0", "8.1", "8.2"]
```

Comment:
I am glad we are able to get all of these into a compatibility test.  Can we set this up to throw a warning/notification on "8.2" but not fail?